app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItem.kt

```diff
@@ -106,7 +106,8 @@ fun MessageItem(
     onSelfDeletingMessageRead: (UIMessage) -> Unit,
     onFailedMessageRetryClicked: (String) -> Unit = {},
     onFailedMessageCancelClicked: (String) -> Unit = {},
-    onLinkClick: (String) -> Unit = {}
+    onLinkClick: (String) -> Unit = {},
+    isFromSearch: Boolean = false
```

Comment:
question: can't we just change the background of the search screen instead of doing this?

----

In case it doesn't work, here's a suggestion:

suggestion: the message item shouldn't care if it's from search or not.

What if now we display this also in some other screen, like in the `Mentions` list or whatever. Then we'd need another `isFromMentions`?

instead of `isFromSearch`, perhaps just adding a `defaultBackgroundColor`.

Something like:

```suggestion
    defaultBackgroundColor: Color = Color.Transparent
```

And further down:

```kotlin
        } else {
            Modifier.background(defaultBackgroundColor)
        }
```

instead of adding an extra condition for the `search` part.

The search caller can call it with the desired `defaultBackgroundColor`.